core/src/main/scripts/importer/validateData.py

```diff
@@ -1984,6 +1985,12 @@ class ClinicalValidator(Validator):
             'datatype': 'STRING'
         },
     }
+
+    # For some attributes, only a warning should be reported when they are in the wrong data file.
+    # This was added to successfully validate old study data.
+    WARNING_ATTRIBUTES = ['PRIMARY_SITE', 'METASTATIC_SITE', 'TUMOR_TISSUE_SITE']
```

Comment:
please add a few words on how it was tested/verified that having these attributes (only) in the wrong file does not affect functionality. And does this mean these were previously set to `logger.error` without a good reason? Or is the original problem solved (e.g. in the frontend)?

Reply:
See https://github.com/cBioPortal/cbioportal/blob/d82c5fc7b40dd1f4a5fcffa67976e150dad4f382/docs/File-Formats.md#custom-columns-in-clinical-data

These properties don't lead issues in the UI, but can make the data incomparable between studies. I tested this in the patient & sample view.

Reply:
What happens when these attributes are in both patient and sample file? What if they have conflicting values? Is sample given priority? Is there a warning during validation? Will the UI be able to handle this?

What happens when the patient has multiple samples, but only patient has the attribute METASTATIC_SITE? Will all samples be labelled metastatic?